lib/bin/lwm2m_carrier/Kconfig

```diff
@@ -204,6 +244,8 @@ config LWM2M_CARRIER_QUEUE_MODE
 	  Configure whether the LwM2M device is to inform the LwM2M server that
 	  it may be disconnected for an extended period of time.
 
+menu "Device info"
```

Comment:
Default value for `LWM2M_CARRIER_DEVICE_TYPE` is not needed because this is also the default value when not set.